br/tests/lightning_tikv_multi_rocksdb/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# Copyright 2020 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+. $cur/../_utils/run_services
+export TIKV_CONFIG="tests/config/tikv-multi-rocksdb.toml"
+restart_services
```

Comment:
How can we ensure whether the multi-rocksdb config takes effect? `restart_services` seems just to restart the process, but not clean up previous data.

Reply:
https://github.com/pingcap/tidb/pull/42243/commits/472d541a3dfa3a3560331149cb84ab378c6005c3

Reply:
I believe TiKV uses the new configuration. But my concern is that since we are not deleting the old data here. If we just restart the process, will there be some configurations that are still the same as before?

Reply:
cleanup_data is called in start_services_impl

Reply:
And multi-rocksdb developers confirmed https://github.com/pingcap/tidb/pull/42243/commits/2c896b62a2d6f71365d40caf9a5e3dc8b0e8204f can check tikv is switched to new configuration

Reply:
Got it.